src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal.tsx

```diff
@@ -24,7 +24,7 @@ export interface CreateNewListValues {
 interface CreateNewListModalProps {
   visible: boolean
   onClose: () => void
-  onComplete: () => void
+  onComplete: (listName: string) => void
```

Comment:
Changes is this file will vanish once I fix merge conflicts with https://github.com/artsy/force/pull/12021